daft/gravitino/README.md

```diff
@@ -0,0 +1,281 @@
+catalog = client.load_catalog("my_catalog")
+print(f"Catalog: {catalog.name}, Type: {catalog.type}, Provider: {catalog.provider}")
+
+# List schemas in a catalog
+schemas = client.list_schemas("my_catalog")
+print(f"Schemas: {schemas}")
+
+# List tables in a schema
+tables = client.list_tables("my_catalog.my_schema")
+print(f"Tables: {tables}")
+```
+
+### Reading Data
+
+#### Using Daft Catalog (Recommended)
+
+```python
+from daft.catalog import Catalog, Table
+from daft.gravitino import GravitinoClient
+
+# Create catalog
+client = GravitinoClient("http://localhost:8090", "my_metalake", username="admin")
+catalog = Catalog.from_gravitino(client)
+
+# Read table using catalog
+df = catalog.read_table("my_catalog.my_schema.my_table")
+df.show()
+
+# Or get table object first
+table = catalog.get_table("my_catalog.my_schema.my_table")
+df = table.read()
+
+# Create table from Gravitino table object
+gravitino_table = client.load_table("my_catalog.my_schema.my_table")
+table = Table.from_gravitino(gravitino_table)
+df = table.read()
+```
+
+#### Using Direct Client
+
+```python
+# Load a table from Gravitino
+table = client.load_table("my_catalog.my_schema.my_table")
+
+# Read with Daft based on table format
+if table.table_info.format.upper().startswith("ICEBERG"):
+    df = daft.read_iceberg(table.table_uri, io_config=table.io_config)
+elif table.table_info.format.upper() == "PARQUET":
+    df = daft.read_parquet(table.table_uri, io_config=table.io_config)
+
+# Display the data
+df.show()
+```
+
+### Loading Tables
+
+```python
+# Load an existing table
+try:
+    table = client.load_table("my_catalog.my_schema.existing_table")
+    print(f"Loaded table: {table.table_info.name}")
+except Exception as e:
+    print(f"Table not found: {e}")
+```
+
+### Working with Filesets
+
+```python
+# Load a fileset
+fileset = client.load_fileset("my_catalog.my_schema.my_fileset")
+
+# Access files in the fileset (example with Parquet files)
+df = daft.read_parquet(f"{fileset.fileset_info.storage_location}/*.parquet",
+                       io_config=fileset.io_config)
+```
+
+## Configuration
+
+### Authentication
+
+The client supports two authentication methods:
+
+1. **Simple Authentication**: Uses username/password or just username
+2. **OAuth2**: Uses bearer token authentication
+
+### Storage Credentials
+
+Gravitino manages storage credentials through table and fileset properties. The client automatically extracts and configures:
+
+- **S3**: Access key, secret key, and session token
+- **Azure**: SAS tokens for blob storage access
+- **GCS**: Service account credentials (planned)
+
+## API Reference
+
+### Daft Catalog Integration
+
+#### Catalog.from_gravitino(client)
+
+Creates a Daft Catalog from a GravitinoClient.
+
+```python
+from daft.catalog import Catalog
+from daft.gravitino import GravitinoClient
+
+client = GravitinoClient("http://localhost:8090", "my_metalake", username="admin")
+catalog = Catalog.from_gravitino(client)
```

Comment:
Thanks for this nice README. Could you please update the docs? This will generate the proper API documentation for the website.

Reply:
sure, will update together.